src/config.json

```diff
@@ -1258,6 +1258,65 @@
   ],
   "blacklist": [
     "xn--opensablog-vmb.com",
+    "lacoste.cat",
+    "a156metamask.com",
+    "appuniswap.org",
+    "aptos-pancakeswap.finance",
+    "autholedgwallaps.org",
+    "authorise-coinbase.com",
+    "binance-exchange.com",
+    "binancewallet.org",
+    "coinbase-account49.dynamic-dns.net",
+    "coinbase-login-auth32.mefound.com",
+    "coinbase-secure-account.com",
+    "coinbase-sign-auth.dynamic-dns.net",
+    "coinbase.com---crypto.space",
+    "coinbaseverify.live",
+    "coinbasewallet.googleystore.com",
```

Comment:
`googleystore.com` is already included so this one is also redundant.
LGTM otherwise - thank you for being on top of this!

Reply:
got it, thanks!